src/pages/tracked-contracts/TrackedContractsList.tsx

```diff
@@ -33,46 +18,48 @@ const TrackedContractsList: React.FC<{}> = () => {
   // have a call from the tracked contract when something changed to let the parent know
   // We should have a modal displaying that there is some current changes to update
   // this is just a safe precaution
-
   useEffect(() => {
-    setChains(mockDataChain as Chain[]);
+    ChainsApi.getAllChains().then((res: any) => {
+      setChains(res.data.chains);
+      // Select the first chain as active
+      setActiveChain(res.data.chains[0]);
+    });
   }, []);
 
-  // const selectChain = (chain: Chain): void => {
-  //   setActiveChain(chain);
-  //   setActiveContracts(chain.tracked_contracts.length ? chain.tracked_contracts : ['No Tracked Contracts']);
-  // };
-
   // Filter out by the search but also keep the current active in the search
   const allChains = chains
     .filter((chain) => chain.name.includes(searchChain.trim().toLocaleLowerCase()) || activeChain?.name === chain.name)
     .map((chain) => (
-      <TrackedChain active={chain.name === activeChain?.name} name={chain.name} onClick={() => setActiveChain(chain)} />
```

Comment:
Thinking about it @jbargu  and your suggestion, I might remove the search and instead have a selector there

<img width="294" alt="Screenshot 2021-04-01 at 16 27 08" src="https://user-images.githubusercontent.com/6785920/113317302-28977200-9307-11eb-8678-0b3b054267e1.png">
Or we could have just one small selectors to select the type 'evm' | 'substrate' ( or both )